plugin/pluginmodel/config.go

```diff
@@ -188,6 +189,12 @@ type APIServerVolume struct {
 	ReadOnly bool   `yaml:"readOnly,omitempty"`
 }
 
+type KubernetesControllerManager struct {
+	Flags ControllerFlags `yaml:"flags,omitempty"`
+}
+
+type ControllerFlags []CommandLineFlag
```

Comment:
I'm fine with removing the dedicated type `APIServerFlags` and just use `[]CommandLineFlag` instead for simplicity!